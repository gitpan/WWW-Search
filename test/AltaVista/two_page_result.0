<html><head>
<title>AltaVista: Simple Query +LSAM +ISI +IB</title>
</head><body>
<font size=+1 face=arial, helvetica>
<a href="/cgi-bin/query?text">AltaVista Search Service</a>
</font>
<br>
<FORM name=mfrm method=GET action="/cgi-bin/query">
<INPUT TYPE=hidden NAME=pg VALUE=q>
<INPUT TYPE=hidden NAME=text VALUE=yes>
<font size=-1><B>Ask AltaVista<sup><small>TM</small></sup> a question.</B>&nbsp;&nbsp;Or enter a few words in
<SELECT NAME=kl><OPTION VALUE=XX SELECTED>any language 
<OPTION VALUE=zh>Chinese
<OPTION VALUE=cs>Czech
<OPTION VALUE=da>Danish
<OPTION VALUE=nl>Dutch
<OPTION VALUE=en>English
<OPTION VALUE=et>Estonian
<OPTION VALUE=fi>Finnish
<OPTION VALUE=fr>French
<OPTION VALUE=de>German
<OPTION VALUE=el>Greek
<OPTION VALUE=he>Hebrew
<OPTION VALUE=hu>Hungarian
<OPTION VALUE=is>Icelandic
<OPTION VALUE=it>Italian
<OPTION VALUE=ja>Japanese
<OPTION VALUE=ko>Korean
<OPTION VALUE=lv>Latvian
<OPTION VALUE=lt>Lithuanian
<OPTION VALUE=no>Norwegian
<OPTION VALUE=pl>Polish
<OPTION VALUE=pt>Portuguese
<OPTION VALUE=ro>Romanian
<OPTION VALUE=ru>Russian
<OPTION VALUE=es>Spanish
<OPTION VALUE=sv>Swedish
</SELECT></font><br><INPUT NAME=q size=50 maxlength=800 VALUE="+LSAM +ISI +IB">
<br><INPUT TYPE=radio NAME=act VALUE=search CHECKED>search <INPUT TYPE=radio NAME=act VALUE=refine>refine <INPUT TYPE=submit VALUE=Submit><br>
 <P><font size=-1>Search - <a href="/cgi-bin/query?pg=aq&what=web&text=yes">Advanced</a> - <a href="/cgi-bin/query?pg=q&what=news&text=yes">Usenet</a></font><BR></FORM>
<font size=-1><b>AltaVista found 18 Web pages for you. </font></b><P><font size=-1><b>Result Pages: </b><b>1</b> &nbsp;<a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bIB&stq=10&c9k">2</a>
&nbsp; <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bIB&stq=10&c9k">[<b>Next &gt;&gt;</b>]</a> <br>
</font><font size=-1><dl><dt><b>1. </b><a href="http://www.isi.edu/ib/example.html"><b>IB Research Plan</b></a><dd>
IB Research Plan. For the experiments in this research, we will make use of traffic generators that emulate WWW browser request streams, to evaluate the...<br><b>URL:</b> <font color=gray>www.isi.edu/ib/example.html<br>
Last modified 13-Jan-97 - page size 6K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fib%2fexample%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>2. </b><a href="http://www.isi.edu/div7/cities/ib/tuner/about.html"><b>About IB</b></a><dd>
About the Multicast Web Tuner. Proxy Benefits: Move files closer to client to improve response speed when files are requested again. Share proxies so that.<br><b>URL:</b> <font color=gray>www.isi.edu/div7/cities/ib/tuner/about.html<br>
Last modified 12-Feb-98 - page size 4K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2fcities%2fib%2ftuner%2fabout%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>3. </b><a href="http://www.isi.edu/~lsam/ib/rmc/investigate.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
IB - Reliable Multicast Transport protocol. AFDP. Dynamically selects from available transmission modes so as to maximize efficiency and throughput while..<br><b>URL:</b> <font color=gray>www.isi.edu/~lsam/ib/rmc/investigate.html<br>
Last modified 12-Feb-98 - page size 3K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7elsam%2fib%2frmc%2finvestigate%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>4. </b><a href="http://www.isi.edu/~lsam/ib/rmc/links.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
IB - Reliable Multicast Transport protocol. Here are a few good links for Multicast Transport Protocols....<br><b>URL:</b> <font color=gray>www.isi.edu/~lsam/ib/rmc/links.html<br>
Last modified 12-Feb-98 - page size 1K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7elsam%2fib%2frmc%2flinks%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>5. </b><a href="http://www.isi.edu/ib/discussion.html"><b>IB Discussion</b></a><dd>
IB Discussion. Response time and bandwidth are key resources in this project. Current Internet user interfaces, such as Web browsers, Gopher and WAIS...<br><b>URL:</b> <font color=gray>www.isi.edu/ib/discussion.html<br>
Last modified 13-Jan-97 - page size 6K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fib%2fdiscussion%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>6. </b><a href="http://www.isi.edu/div7/ib/plan.html"><b>IB Research Plan</b></a><dd>
IB Research Plan. For the experiments in this research, we will make use of traffic generators that emulate WWW browser request streams, to evaluate the...<br><b>URL:</b> <font color=gray>www.isi.edu/div7/ib/plan.html<br>
Last modified 13-Jan-97 - page size 4K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2fib%2fplan%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>7. </b><a href="http://www.isi.edu/~lsam/ib/index.html"><b>About IB</b></a><dd>
Intelligent Bandwidth. About Intelligent Bandwidth. Research Projects. People. Related Work. Page maintainer: Joe Touch Last modified: Thu Jan 15 13:30:29.<br><b>URL:</b> <font color=gray>www.isi.edu/~lsam/ib/index.html<br>
Last modified 15-Jan-98 - page size 2K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7elsam%2fib%2findex%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>8. </b><a href="http://www.isi.edu/~lsam/ib/people.html"><b>IB Research Plan</b></a><dd>
IB People. Joe Touch (project leader) Amy S. Hughes. Anand Oswal. Page maintainer: Joe Touch Last modified: Thu Jan 15 13:31:54 1998 Copyright © 1996 by...<br><b>URL:</b> <font color=gray>www.isi.edu/~lsam/ib/people.html<br>
Last modified 15-Jan-98 - page size 2K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7elsam%2fib%2fpeople%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>9. </b><a href="http://www.isi.edu/div7/lsam/ib/rmc/about.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
IB - Reliable Multicast Transport protocol. Goals. To use idle bandwidth to pre-load caches with objects that might be of interest in the future. The...<br><b>URL:</b> <font color=gray>www.isi.edu/div7/lsam/ib/rmc/about.html<br>
Last modified 12-Feb-98 - page size 2K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2flsam%2fib%2frmc%2fabout%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>10. </b><a href="http://www.isi.edu/div7/lsam/ib/rmc/design.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
IB - Reliable Multicast Transport protocol. Reliability Mechanism: The idea of multicasting NACks is based on the fact that multicast losses are...<br><b>URL:</b> <font color=gray>www.isi.edu/div7/lsam/ib/rmc/design.html<br>
Last modified 12-Feb-98 - page size 4K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2flsam%2fib%2frmc%2fdesign%2ehtml&language=en">Translate</a>&nbsp;]</dl>
</font><font size=-1><b>Result Pages: </b><b>1</b> &nbsp;<a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bIB&stq=10&c9k">2</a>
&nbsp; <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bIB&stq=10&c9k">[<b>Next &gt;&gt;</b>]</a> <br>
</font><font size=-2>word count: LSAM: 893; ISI: 130445; IB: 169417<BR></font><P>
<br>
<font size=-2 face=arial, helvetica>
<a href="/cgi-bin/query?text">AltaVista Home</a> | 
<a href="/av/content/help.htm">Help</a> | 
<a href="/av/content/questions.htm">Feedback</a>
<br> 
Copyright | 
<a href="/av/content/disclaimer.htm">Disclaimer</a> | 
<a href="/av/content/privacy.htm">Privacy</a>
<br>
<a href="/av/content/about.htm">About AltaVista</a> | 
<a href="/cgi-bin/query?pg=tmpl&v=pref.html">Set your Preferences</a> | 
<a href="/av/content/addurl.htm">Add a Page</a> | 
<a href="/cgi-bin/query">Graphic Version</a>
</font>
</BODY></HTML>
