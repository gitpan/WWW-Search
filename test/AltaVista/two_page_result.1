<html><head>
<title>AltaVista: Simple Query +LSAM +ISI +work</title>
</head><body>
<a href="/cgi-bin/query?pg=q&text=yes"><b>AltaVista Search</b></a><br>
<FORM name=mfrm method=GET action="/cgi-bin/query">
<INPUT TYPE=hidden NAME=pg VALUE=q>
<INPUT TYPE=hidden NAME=stq VALUE=10>
<INPUT TYPE=hidden NAME=text VALUE=yes>
<font size=-1 face="arial, helvetica"><B>Search <SELECT NAME=what><OPTION VALUE=web SELECTED>the Web<OPTION VALUE=news >Usenet</SELECT>
for documents in <SELECT NAME=kl><OPTION VALUE=XX SELECTED>any language 
<OPTION VALUE=zh >Chinese
<OPTION VALUE=cz >Czech
<OPTION VALUE=da >Danish
<OPTION VALUE=nl >Dutch
<OPTION VALUE=en >English
<OPTION VALUE=et >Estonian
<OPTION VALUE=fi >Finnish
<OPTION VALUE=fr >French
<OPTION VALUE=de >German
<OPTION VALUE=el >Greek
<OPTION VALUE=he >Hebrew
<OPTION VALUE=hu >Hungarian
<OPTION VALUE=is >Icelandic
<OPTION VALUE=it >Italian
<OPTION VALUE=ja >Japanese
<OPTION VALUE=ko >Korean
<OPTION VALUE=lv >Latvian
<OPTION VALUE=lt >Lithuanian
<OPTION VALUE=no >Norwegian
<OPTION VALUE=pl >Polish
<OPTION VALUE=pt >Portuguese
<OPTION VALUE=ro >Romanian
<OPTION VALUE=ru >Russian
<OPTION VALUE=es >Spanish
<OPTION VALUE=sv >Swedish
</SELECT></B></font><br><INPUT NAME=q size=65 maxlength=800 wrap=virtual VALUE="+LSAM +ISI +work"><br><INPUT TYPE=radio NAME=act VALUE=search CHECKED>search <INPUT TYPE=radio NAME=act VALUE=refine>refine <INPUT TYPE=submit VALUE=Submit>
 <BR><a href="/av/content/help_simple.htm">Help</a> . <a href="/cgi-bin/query?pg=tmpl&v=pref.html&text=yes">Preferences</a> . <a href="/cgi-bin/query?pg=q&text=yes">New Search</a> . <a href="/cgi-bin/query?pg=aq&text=yes">Advanced Search</a><br><BR><font size=-1 face="arial, helvetica"><b>13</b> documents match your query.</font><BR><dl>
<dt><font size=-1><b>11. </b></font><a href="http://gost.isi.edu/~johnh/CHANGES/"><strong>Changes to John Heidemann's Web Pages</strong></a><dd>Changes to John Heidemann's Web Pages. A list of changes to my web pages, in reverse chronological order: 2 May 1997. USC-to-ISI directions. 1 May 1997....<br><cite><a href="http://gost.isi.edu/~johnh/CHANGES/">http://gost.isi.edu/~johnh/CHANGES/</a> - size 9K - 2-May-97 - English</cite><br>
<P><dt><font size=-1><b>12. </b></font><a href="http://andrew2.andrew.cmu.edu/rfc/rfc2140.html"><strong>RFC 2140</strong></a><dd>Network Working Group J. Touch Request for Comments: 2140 ISI Category: Informational April 1997. Page 1. TCP Control Block Interdependence. Status of...<br><cite><a href="http://andrew2.andrew.cmu.edu/rfc/rfc2140.html">http://andrew2.andrew.cmu.edu/rfc/rfc2140.html</a> - size 25K - 3-May-97 - English</cite><br>
<P><dt><font size=-1><b>13. </b></font><a href="http://www12.w3.org/Protocols/HTTP/Issues/touch.html"><strong>No Title</strong></a><dd>View: Browse HTML Browse Raw Text From: touch@ISI.EDU Date: Thu, 13 Mar 1997 13:11:48 -0800 To: jg@pa.dec.com, fielding@ics.uci.edu Cc: touch@ISI.EDU,...<br><cite><a href="http://www12.w3.org/Protocols/HTTP/Issues/touch.html">http://www12.w3.org/Protocols/HTTP/Issues/touch.html</a> - size 2K - 4-Apr-97 - English</cite><br>
<P></dl>
<b>Tip: </b>

The <b>word count</b>, located below the results, indicates how many times
a term appears in the entire Web.  
Use it as a spelling or usage checker: type in both
spellings for a word, and see which one wins.
<br><BR><font size=-1 face="arial, helvetica"><font size=-1>Word count: LSAM: 92; ISI: 41752<BR>Ignored:    work: 8017848<BR></font>
</font><P><a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=0">[Prev]</a> <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=0">1 </a>2 <br>
<HR><font face="arial, helvetica" size=-1><a href="/av/content/av_network.htm">Our Network</a> | <a href="/av/content/addurl.htm">Add/Remove URL</a> | <a href="/av/content/questions.htm">Feedback</a> | <a href="/av/content/help.htm">Help</a><br><a href="http://www.doubleclick.net/jumps/jump1217.htm">Advertising Info</a> | <a href="/av/content/about.htm">About AltaVista</a> | <a href="/av/content/jobs.htm">Jobs</a> | <a href="/cgi-bin/query?pg=q">Graphics</a></font><HR><a href="/av/content/disclaimer.htm">Disclaimer</a> | <a href="/av/content/privacy.htm">Privacy Statement</a><br><a href="http://www.digital.com"><b>DIGITAL</b></a> Copyright 1997. All Rights Reserved</FORM>
</BODY></HTML>
