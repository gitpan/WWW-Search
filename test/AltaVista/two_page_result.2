<html><head>
<title>AltaVista: Simple Query +LSAM +ISI +work</title>
</head><body>
<font size=+1>
<a href="/cgi-bin/query?text">AltaVista Search Service</a>
</font> - 
<a href="/cgi-bin/query?text">Search</a> - 
<a href="/av/content/zones.html">Zones</a> - 
<a href="/av/content/services.html">Services</a> - 
<a href="/av/content/help.htm">Help</a> - 
<a href="/av/content/questions.htm">Feedback</a>
<br>
<FORM name=mfrm method=GET action="/cgi-bin/query">
<INPUT TYPE=hidden NAME=pg VALUE=q>
<INPUT TYPE=hidden NAME=text VALUE=yes>
<font size=-1><B>Search the Web for documents in</B> <SELECT NAME=kl><OPTION VALUE=XX SELECTED>any language 
<OPTION VALUE=zh>Chinese
<OPTION VALUE=cs>Czech
<OPTION VALUE=da>Danish
<OPTION VALUE=nl>Dutch
<OPTION VALUE=en>English
<OPTION VALUE=et>Estonian
<OPTION VALUE=fi>Finnish
<OPTION VALUE=fr>French
<OPTION VALUE=de>German
<OPTION VALUE=el>Greek
<OPTION VALUE=he>Hebrew
<OPTION VALUE=hu>Hungarian
<OPTION VALUE=is>Icelandic
<OPTION VALUE=it>Italian
<OPTION VALUE=ja>Japanese
<OPTION VALUE=ko>Korean
<OPTION VALUE=lv>Latvian
<OPTION VALUE=lt>Lithuanian
<OPTION VALUE=no>Norwegian
<OPTION VALUE=pl>Polish
<OPTION VALUE=pt>Portuguese
<OPTION VALUE=ro>Romanian
<OPTION VALUE=ru>Russian
<OPTION VALUE=es>Spanish
<OPTION VALUE=sv>Swedish
</SELECT></font><br><INPUT NAME=q size=50 maxlength=800 VALUE="+LSAM +ISI +work">
<br><INPUT TYPE=radio NAME=act VALUE=search CHECKED>search <INPUT TYPE=radio NAME=act VALUE=refine>refine <INPUT TYPE=submit VALUE=Submit><br>
 <font face=arial size=-2><b>Tip: </b>


To find all pages pointing to a URL, try: <b>link:mypage.html</b>.
<a href="/av/content/help.htm#simple">More tips</a></font><br>
<P><font size=-2>Search - <a href="/cgi-bin/query?pg=aq&text=yes">Advanced</a> - <a href="http://altavista.switchboard.com/bin/cgiqa.dll?MG=&MEM=212">Find a Person</a> - <a href="http://altavista.switchboard.com/bin/cgidir.dll?MG=&MEM=212">Find a Business</a> - <a href="http://altavista.looksmart.com/altavista.html">Browse Categories</a> - <a href="/cgi-bin/query?pg=q&what=news&text=yes">Search Usenet</a></font><BR></FORM>
<font size=-1><b>87</b> matches were found. </font><P><font size=-1>Pages: <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=10&c9k">[<b>&lt;&lt;</b>]</a>  <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=0&c9k">1</a>
 <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=10&c9k">2</a>
 <b>3</b> <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=30&c9k">4</a>
 <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=40&c9k">5</a>
 <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=50&c9k">6</a>
 <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=60&c9k">7</a>
 <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=70&c9k">8</a>
 <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=80&c9k">9</a>
<a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=30&c9k">[<b>&gt;&gt;</b>]</a> <P>
</font><font size=-1><dl>
<dt><b>21. </b><a href="http://www.isi.edu/~johnh/SOFTWARE/APACHE/index.html"><b>LSAM Apache Patches</b></a><dd>
<font size=-2>[<b>URL:</b> www.isi.edu/~johnh/SOFTWARE/APACHE/index.html]</font><br>
LSAM Apache Patches. As part of my work with ISI's LSAM project we have found some optimizations in http-server performance. We describe these problems in.<br><font size=-2>Last modified 27-Mar-98 - page size 3K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7ejohnh%2fSOFTWARE%2fAPACHE%2findex%2ehtml&language=en">Translate</a> ]</font><P>
<dt><b>22. </b><a href="http://www.isi.edu/lsam/publications/index.html"><b>LSAM Publications</b></a><dd>
<font size=-2>[<b>URL:</b> www.isi.edu/lsam/publications/index.html]</font><br>
LSAM Publications. LSAM general: "The LSAM Proxy Cache - a Multicast Distributed Virtual Cache" (project overview) J. Touch, (submitted to the 3rd...<br><font size=-2>Last modified 3-Apr-98 - page size 4K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2flsam%2fpublications%2findex%2ehtml&language=en">Translate</a> ]</font><P>
<dt><b>23. </b><a href="http://xent.ics.uci.edu/FoRK-archive/jan98/0429.html"><b>FoRK Archive: LSAM: Large-Scale Active Middleware (formerly Cities Online)</b></a><dd>
<font size=-2>[<b>URL:</b> xent.ics.uci.edu/FoRK-archive/jan98/0429.html]</font><br>
LSAM: Large-Scale Active Middleware (formerly Cities Online) Rohit Khare (rohit@uci.edu) Tue, 03 Feb 1998 00:05:42 -0800. Messages sorted by: [ date ][...<br><font size=-2>Last modified 3-Feb-98 - page size 9K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fxent%2eics%2euci%2eedu%2fFoRK%2darchive%2fjan98%2f0429%2ehtml&language=en">Translate</a> ]</font><P>
<dt><b>24. </b><a href="http://www.isi.edu/div7/lsam/about/"><b>About LSAM</b></a><dd>
<font size=-2>[<b>URL:</b> www.isi.edu/div7/lsam/about/]</font><br>
About LSAM. What is LSAM. LSAM is developing middleware infrastructure to support scalable distributed information services. This is embodied in the LSAM..<br><font size=-2>Last modified 26-Mar-98 - page size 9K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2flsam%2fabout%2f&language=en">Translate</a> ]</font><P>
<dt><b>25. </b><a href="http://www.isi.edu/div7/lsam/publications/index.html"><b>LSAM Publications</b></a><dd>
<font size=-2>[<b>URL:</b> www.isi.edu/div7/lsam/publications/index.html]</font><br>
LSAM Publications. LSAM general: "The LSAM Proxy Cache - a Multicast Distributed Virtual Cache" (project overview) J. Touch, (submitted to the 3rd...<br><font size=-2>Last modified 3-Apr-98 - page size 4K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2flsam%2fpublications%2findex%2ehtml&language=en">Translate</a> ]</font><P>
<dt><b>26. </b><a href="http://www.isi.edu/div7/cities/about/performance.html"><b>About LSAM Performance Analysis</b></a><dd>
<font size=-2>[<b>URL:</b> www.isi.edu/div7/cities/about/performance.html]</font><br>
About LSAM Performance Analysis. Distributed Object Performance. LSAM is examining protocol extensions to improve the performance of distributed object...<br><font size=-2>Last modified 16-Dec-97 - page size 6K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2fcities%2fabout%2fperformance%2ehtml&language=en">Translate</a> ]</font><P>
<dt><b>27. </b><a href="http://www.isi.edu/lsam/ib/related-work.html"><b>LSAM Related Research</b></a><dd>
<font size=-2>[<b>URL:</b> www.isi.edu/lsam/ib/related-work.html]</font><br>
nbsp;Web Latency Reduction. &nbsp;&nbsp;&nbsp; Features Description. &nbsp; Anticipation. Configuration. Cache Organization. Intermediate Proxies. User...<br><font size=-2>Last modified 15-Jan-98 - page size 6K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2flsam%2fib%2frelated%2dwork%2ehtml&language=en">Translate</a> ]</font><P>
<dt><b>28. </b><a href="http://www.isi.edu/div7/cities/about/replication.html"><b>About LSAM Replication</b></a><dd>
<font size=-2>[<b>URL:</b> www.isi.edu/div7/cities/about/replication.html]</font><br>
About LSAM Replication. Why Replication? Although the Web is widely used today, for it to be truly effective part of daily work, it must provide both...<br><font size=-2>Last modified 6-Jan-97 - page size 4K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2fcities%2fabout%2freplication%2ehtml&language=en">Translate</a> ]</font><P>
<dt><b>29. </b><a href="http://www.isi.edu/lsam/ib/features.html"><b>LSAM Features</b></a><dd>
<font size=-2>[<b>URL:</b> www.isi.edu/lsam/ib/features.html]</font><br>
nbsp;Feature Descriptions. &nbsp;Anticipation. &nbsp;&nbsp;&nbsp; Client Prefetching. &nbsp;&nbsp;&nbsp; Client determines which pages to load in the...<br><font size=-2>Last modified 15-Jan-98 - page size 7K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2flsam%2fib%2ffeatures%2ehtml&language=en">Translate</a> ]</font><P>
<dt><b>30. </b><a href="http://www.isi.edu/lsam/about/"><b>About LSAM</b></a><dd>
<font size=-2>[<b>URL:</b> www.isi.edu/lsam/about/]</font><br>
About LSAM. What is LSAM. LSAM is developing middleware infrastructure to support scalable distributed information services. This is embodied in the LSAM..<br><font size=-2>Last modified 26-Mar-98 - page size 9K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2flsam%2fabout%2f&language=en">Translate</a> ]</font><P>
</dl>
</font><font size=-1>Pages: <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=10&c9k">[<b>&lt;&lt;</b>]</a>  <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=0&c9k">1</a>
 <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=10&c9k">2</a>
 <b>3</b> <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=30&c9k">4</a>
 <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=40&c9k">5</a>
 <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=50&c9k">6</a>
 <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=60&c9k">7</a>
 <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=70&c9k">8</a>
 <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=80&c9k">9</a>
<a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=30&c9k">[<b>&gt;&gt;</b>]</a> <P>
</font><font size=-2>word count: LSAM: 909; ISI: 176948; work: 31015219<BR></font><P>
<br>
<font size=-2 face=arial>
<a href="/cgi-bin/query?text">Search</a> | 
<a href="/av/content/zones.html">Zones</a> | 
<a href="/av/content/services.html">Services</a> | 
<a href="/av/content/help.htm">Help</a> | 
<a href="/av/content/questions.htm">Feedback</a>
<br> 
Copyright (c) 1995-98 
<a href="http://www.digital.com">Digital Equipment Corporation</a> | 
<a href="/av/content/disclaimer.htm">Disclaimer</a> | 
<a href="/av/content/privacy.htm">Privacy</a>
<br>
<a href="/av/content/about.htm">About AltaVista</a> | 
<a href="/cgi-bin/query?pg=tmpl&v=pref.html">Set your Preferences</a> | 
<a href="/av/content/addurl.htm">Add a Page</a> | 
<a href="/cgi-bin/query">Graphic Version</a>
</font>
</BODY></HTML>
