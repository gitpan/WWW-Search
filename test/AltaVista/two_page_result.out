http://gost.isi.edu/~johnh/SOFTWARE/
http://gost.isi.edu/~johnh/WORK/
http://gost.isi.edu/~johnh/SOFTWARE/APACHE/
http://gost.isi.edu/people/johnh/SOFTWARE/APACHE/apache_1.0_isi_mmap_960524.patch
http://www.isi.edu/~johnh/home_page.html
http://gost.isi.edu/people/johnh/SOFTWARE/APACHE/apache_1.0_isi_mmap_961202.patch
http://www.isi.edu/~katia/
http://gost.isi.edu/brian/cities/
http://gost.isi.edu/brian/resume/work.html
http://www-uk.hpl.hp.com/people/ange/archives/http-wg-archive/0258.html
http://gost.isi.edu/~johnh/CHANGES/
http://andrew2.andrew.cmu.edu/rfc/rfc2140.html
http://www12.w3.org/Protocols/HTTP/Issues/touch.html
